ONSdigital/dp-frontend-feedback-controller
# fix/remove extra v1

### What

Updated `dp-feedback-api` to remove extra `v1`

### How to review

Sense check
Or run locally
- Port forward api router to sandbox
- Run `dp-design-system`
- Pull and run this branch `make debug ENABLE_FEEDBACK_API=true`
- Submit the feedback form and check the email in sandbox

### Who can review

!me
